Donation-platform/src/main/java/com/google/FindTrendingCharities.java

```diff
@@ -159,4 +159,15 @@ private double getTagTrendingScore(Collection<Tag> tags) throws Exception {
     }
     return (sumScores / numTags);
   }
+
+    public Collection<Tag> getTagsDb() throws Exception{
+        Collection<Tag> tags= db.getAllTags();
```

Comment:
Nitpick: consistent spacing

Reply:
thanks for pointing it out. will adjust it.